src/style.css

```diff
@@ -1,3 +1,98 @@
+/* stylelint-disable string-quotes */
+@font-face {
+  font-family: 'Roboto';
+  src: url('fonts/Roboto-Medium.ttf') format('woff2');
+  font-weight: 500;
+  font-style: normal;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: 'Roboto', sans-serif;
+}
+
+.header {
+  display: flex;
+  height: 60px;
+  background-color: #fff;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)